src/methods.md

```diff
@@ -25,3 +25,17 @@ fn main() {
 }
 ```
 
+<details>
+
+Key Points:
+* It can be helpful to introduce methods by comparing them to functions.
+  * Methods are called on an instance of a struct, the first parameter represents the instance as `self`.
```

Comment:
Not necessarily a struct, they can be implemented for any type (e.g. an enum or a primitive).

Reply:
updated, what do you think

Reply:
I've reworded it in a way that I think is clearer.

Reply:
LGTM